centaur/src/main/scala/centaur/test/Test.scala

```diff
@@ -750,8 +749,7 @@ object Operations extends StrictLogging {
         }
       }
 
-      val jsonDiff = diff(expected: JsValue, actual: JsValue).toJson.prettyPrint
-      IO.raiseError(CentaurTestException(s"Error during $testType metadata comparison. Diff (changes to mutate from 'expected' to 'actual'): $jsonDiff", workflow, submittedWorkflow))
+      IO.raiseError(CentaurTestException(s"Error during $testType metadata comparison. Expected: $expected Actual: $actual", workflow, submittedWorkflow))
```

Comment:
So the JSON diff doesn't include things in expected that AREN'T in actual?

Reply:
I'm sorry, I can't wrap my head around this double negative. But you can [play around with the diff concept online](https://json-patch-builder-online.github.io/).

Maybe this answers your question?

Source
```
{
    "status":"Running",
    "foo":"bar"
}
```

Target
```
{
    "status":"Done",
    "woot":"ftw"
}
```

Diff
```
[
    {
        "op": "remove",
        "path": "/foo"
    },
    {
        "op": "replace",
        "path": "/status",
        "value": "Done"
    },
    {
        "op": "add",
        "path": "/woot",
        "value": "ftw"
    }
]
```

The diff identifies that `"status"` changed but does not mention `"Running"`.

Reply:
Thanks, very helpful.